mw-webapp/src/component/input/Input.tsx

```diff
@@ -69,16 +69,19 @@ interface InputProps {
  * Input component
  */
 export const Input = (props: InputProps) => {
+  const refInput = useRef() as MutableRefObject<HTMLInputElement>;
 
   /**
    * Event handler for the input change event
    */
   const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
     props.onChange(event.target.value);
+    refInput.current.value = "";
   };
 
   return (
     <input
+      ref={refInput}
```

Comment:
Does it possible to solve this issue with no refs?

Reply:
I tried to solve this using the parseInt method, but the fact is that when we focus, we have the values ​​in the input from the database, and when you enter they are simply added since the entered values ​​have not yet entered the database.

Reply:
This link should help with this issue

https://github.com/facebook/react/issues/9402